src/lib/utilities/to-duration.ts

```diff
@@ -121,12 +121,14 @@ export const fromDate = (
 };
 
 export const fromSeconds = (seconds: string): Duration | undefined => {
-  const milliseconds = parseInt(seconds) * 1000;
+  const parsedSeconds = parseInt(seconds);
```

Comment:
Today I learned that you can parseInt('1235s'). I did not think you could have a letter in the string.

Reply:
As long as it starts with the integer, `parseInt('s1235')` returns a `NaN` 😅